scripts/analytics/benchmarks.py

```diff
@@ -139,6 +140,93 @@ def get_log_counts(users):
 
 
 def main():
+
+    number_users = User.find().count()
```

Comment:
The origin script already contain the function to get user count. This is unnecessary.